controllers/graphController.js

```diff
@@ -141,6 +140,12 @@ GraphController.prototype.embed = function(req, res, next) {
 	}).catch(next)
 }
 
+// "moon-test-one" -> "Moon Test One"
+function convNameToHumanReadable(name) {
```

Comment:
well, this replaces dashes with spaces, so we may decide to use it for something other than names at some other point..